cli/cli_connect.go

```diff
@@ -102,7 +102,11 @@ func (c *cmd) Connect(ctx *cli.Context) error {
 		case internal.CodeTokenRenewError:
 			rpcErr = errors.New(client.AccountTokenRenewError)
 		case internal.CodeAccountExpired:
-			c.browseToSubscriptionPage()
+			c.browseToSubscriptionPage(client.SubscriptionURL, client.SubscriptionDedicatedIPURLLogin)
```

Comment:
shouldn't instead of `SubscriptionDedicatedIPURLLogin` be `SubscriptionURLLogin`?